Cleanup code

Cousnouf opened the pull request.

dutoitc reviewed (changes requested):
Merci de supprimer du commit ce qui est refmon

Commit:
Clean up after inspection
Renaming CopperServices to CopperController + use of @RequestParam to use default values (spring alignment)
